public void recover(String fname) throws InterruptedException, IOException {
  chkptInner(fname,false);
}
